packages/devtools_app/lib/src/profiler/cpu_profile_model.dart

```diff
@@ -203,10 +233,19 @@ class CpuProfileData {
         if (profileMetaData?.time?.duration != null)
           timeExtentKey: profileMetaData.time.duration.inMicroseconds,
         stackFramesKey: stackFramesJson,
-        traceEventsKey: stackTraceEvents,
+        traceEventsKey: cpuSamples.map((sample) => sample.json).toList(),
       };
 
   bool get isEmpty => profileMetaData.sampleCount == 0;
+
+  @visibleForTesting
+  Map<String, dynamic> get stackFramesJson {
```

Comment:
nit: prefer Map<String, Object> so we get more static checking.

Reply:
done